WordPress/wordpress-playground
# Add PWA support

## Motivation for the change, related issues

We want to [make Playground available offline](https://github.com/WordPress/wordpress-playground/issues/133).
This PR is a first step and adds support for installing Playground as a PWA. Offline support will be implemented separately. 

## Implementation details

The PR adds a `manifest.json` file and loads it on the Playground website. This allows browsers to install Playground as a PWA

## Testing Instructions (or ideally a Blueprint)

- Checkout this branch
- Open Playground in [a browser that supports PWA installing](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Guides/Making_PWAs_installable#browser_support)
- Confirm that you can install Playground as a PWA
